src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -26,15 +29,15 @@ volatile size_t          sched_cur;
 /* Macro for sinv back to booter from new component */
 enum
 {
-	BOOT_SINV_CAP = round_up_to_pow2(BOOT_CAPTBL_FREE + CAP32B_IDSZ, CAPMAX_ENTRY_SZ)
+	BOOT_SINV_CAP = round_up_to_pow2(BOOT_CAPTBL_FREE + CAP32B_IDSZ, CAPMAX_ENTRY_SZ),
```

Comment:
Is this booter invocation capability required for every component? If it is, should probably move it before `BOOT_CAPTBL_FREE`. 
Also, use the `__captbl_cap2sz` for the size of a capability type.